src/Cli.Tests/EndToEndTests.cs

```diff
@@ -119,7 +119,7 @@ public void TestInitForCosmosDBPostgreSql()
     public void TestInitializingRestAndGraphQLGlobalSettings()
     {
         string[] args = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--database-type", "mssql", "--rest.path", "/rest-api",
-                          "--rest.disabled", "--graphql.path", "/graphql-api" };
+                          "--rest.base-route", "/rest-route", "--rest.disabled", "--graphql.path", "/graphql-api" };
```

Comment:
There might have been some misunderstanding, here's some clarification:

- the `base-route` feature is applicable to the entire runtime - both graphql and rest. So, it needs to be under the runtime section, since when accessed from SWA its `/data-api/rest/*` and `/data-api/graphql`
- Today, SWA does not allow to customize this path, and plans to send us the base route through the configuration endpoint V2 [parameters](https://github.com/Azure/data-api-builder/blob/2e8d1bb8b7d83bbe2a90472c2ccdc8acb87fa5ec/src/Service/Controllers/ConfigurationController.cs#L130). As such it should be honored by the engine.
- SWA will send it as part of system configuration so it stays customer hidden and SWA-CLI will not provide a way today to customize it, may be it will in future, so we can let ability to customize it through DAB CLI